### PR TITLE
Remove completely empty columns/rows from manifest dataframe

### PR DESCRIPTION
**Pull request by sujaypatil96**
Added utility function `trim_commas_df ` (to `df_utils`) to remove completely empty columns and rows from a manifest file. This is an important pre-processing step (method) which can be run on the metadata (manifest) dataframe before we use it for further processing.

Note: This is a fix to resolve issue #376. I've also tested this new backend with an instance of the DC App, and it works. It doesn't "time out" like the previous behaviour.

I'm thinking if I should push an update to the `main` branch as well, so @xdoan can integrate the fix? I'll create a branch called `main-trailiing-commas-fix`. @xdoan can test with that branch and see if the DC App works properly. If yes, then we can merge to `main` as well.

### CODE REVIEW

File: schematic/utils/df_utils.py
Comment (mattfazza) [inline schematic/utils/df_utils.py:72]: It's a bummer they dropped the support to do both of these at once.  In the past you could pass a tuple for the axis.

**Comment by mattfazza** [comment]
I happened to come across the PR on slack so I looked at it - hope that's ok!  I understand you marked Milen as a reviewer, so I'm not approving because of that.  I didn't catch anything wrong with it though! ✅

**Comment by sujaypatil96** [comment]
Thanks for the review @mattfazza.👍  Yup, @milen-sage has write access to the `develop` and `main` branch of this repo, so he'll need to approve any PRs before merging. I'll add you as a reviewer on other PRs (that I/we'll be issuing in the future), if you're interested in the latest updates to the codebase.

File: schematic/utils/df_utils.py
Comment (milen-sage) [reply schematic/utils/df_utils.py:72]: agreed; not sure if it was for readability or to comply with general refactor of interfaces to lower level routines.

**Review by milen-sage — APPROVED** [review]

**Comment by milen-sage** [comment]
Thanks @mattfazza - appreciated! It's always nice to have more eyes go through code. 

[Disclaimer - to be read at a drug commercial speed: package still in development; release protocols not set and code style not unified; CI/CD not included. Side-effects may include dizziness, loss of brain cells, and unanticipated hiccups; experience of occasional nausea and night sweats is normal.]

**Pull request merged**
